orchestrators/FinanceOperations/tests/Unit/Rules/CostCenterActiveRuleTest.php

```
@@ -4,402 +4,144 @@
-                    }
-                    public function getName(): string {
-                        return 'CC';
-                    }
-                };
+        return new class($entries) implements CostCenterQueryInterface {
+            /**
+             * @param array<string, CostCenterRuleViewInterface> $entries
+             */
+            public function __construct(private array $entries) {}
+
+            public function find(string $tenantId, string $costCenterId): ?CostCenterRuleViewInterface
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantId'. (undefined)

(UnusedFormalParameter)